static/specification.html

```diff
@@ -93,34 +105,220 @@
+        <h3>
+          Monetizing media
+        </h3>
+        <p>
+          The following example shows how to monetize various types of media
+          using different [=payment end-points=].
+        </p>
+        <aside class="example">
+          <pre class="html">
+            &lt;video src="myvideo" onmonetization="sayThanks(this)"&gt;
+              &lt;link rel="monetization" href="https://example.com/video/pay"&gt;
+            &lt;/video&gt;
+
+            &lt;audio src="music.mp3" onmonetization="sayThanks(this)"&gt;
+              &lt;link rel="monetization" href="https://example.com/audio/pay"&gt;
+            &lt;/audio&gt;
+
+            &lt;picture srcset="cat.jpeg" onmonetization="sayThanks(this)"&gt;
+              &lt;link rel="monetization" href="https://example.com/images/pay"&gt;
+            &lt;/picture&gt;
+          </pre>
+        </aside>
+      </section>
+    </section>
+    <section id="model">
+      <h2>
+        Model
       </h2>
       <dl>
         <dt>
-          <dfn data-lt="provider">Web Monetization provider</dfn>:
+          <dfn data-lt="monetized">Monetization</dfn>:
+        </dt>
+        <dd>
+          A payment made by a user to a website, facilitated through a
+          [=monetization provider=].
+        </dd>
+        <dt>
+          <dfn data-lt="provider">Monetization provider</dfn>:
         </dt>
         <dd>
-          The party making payments on behalf of the user. A Web Monetization
-          provider uses the [[[SPSP]]] (<abbr title=
-          "Simple Payment Setup Protocol">SPSP</abbr>) on top of the
-          [[[Interledger]]] (<abbr title=
-          "Interledger Protocol">ILP</abbr>)/[[[STREAM]]] to provide a
-          high-level way to send money and data.
+          The party making payments on behalf of the user. A monetization
+          provider leverages the [[[Interledger]]] suite of protocols and
+          technologies (e.g., [[Open Payments]] based on [[STREAM]]) to provide
+          a high-level way to pay a [=monetization receiver=].
         </dd>
         <dt>
-          <dfn>Web Monetization receiver</dfn>:
+          <dfn>Monetization receiver</dfn>:
         </dt>
         <dd>
-          The party receiving payments on behalf of the website.
+          The party receiving payments on behalf of the website, whose details
+          are provided by a [=payment end-point=].
         </dd>
         <dt>
           <dfn>Payment end-point</dfn>:
         </dt>
         <dd>
-          A [=URL=] to a conforming HTTP <a data-cite="SPSP#specification">SPSP
-          end-point</a>.
+          A [=URL=] to a <a data-cite="Open Payments#specification">Open
+          Payments account end-point</a> (i.e., a JSON resource containing
+          details that facilitate payment to an account).
+        </dd>
+        <dt>
+          <dfn>Payment session</dfn>
+        </dt>
+        <dd>
+          An session between a [=monetization provider=] and [=monetization
```

Comment:
What are your thoughts on sessions lasting beyond a tag configuration (page lifetime at most) ? 
Consider a user that has automatic payments turned off, and only likes to tip. 
He tips $5, but then on reloads he's presented with ads. 
Or on a mobile user with an agent that likes to pay big amounts periodically. 
Leave the session to be created using cookies and stitching together a heap of incoming payments / outgoing payments ?